src/app/frontend/deploy/createnamespace_controller.js

```diff
@@ -22,10 +22,11 @@ export default class NamespaceDialogController {
    * @param {!md.$dialog} $mdDialog
    * @param {!angular.$log} $log
    * @param {!angular.$resource} $resource
+   * @param {!Object} errorDialog
```

Comment:
{!./....../errordialog_service.ErrorDialog} is a default export. So it is still not working.

See also: https://github.com/kubernetes/dashboard/issues/159

Currently, we use default export a lot and I thought I should not change this in one place. If we change, we should change everywhere, as suggested by bryk.